docs/api/data-usage-api.md

```diff
@@ -34,10 +34,10 @@ Get the data usage of a company in a month, grouped by SIM card groups.
 
 **Response Payload**
 
-| Field        | Type    | Description                | Required |
-| ------------ | ------- | -------------------------- | -------- |
-| year         | Integer | Year to see data usage in | Yes       |
-| month        | String  | Month, in text, to see consumption for                  | Yes       |
+| Field        | Type    | Description                              | Required |
+| ------------ | ------- | -----------------------------------------| -------- |
+| year         | Integer | Year to see data usage in                | Yes      |
+| month        | String  | Month, in text, to see consumption for   | Yes      |
 
 List([DataUsageBySimCardCollections](/general-information/data-types/#DataUsageBySimCardCollections))
```

Comment:
#DataUsageBySimCardCollections must be in lower case. Else the link to section will not work properly.